l3build/l3build.lua

```diff
@@ -1862,7 +1863,7 @@ function doc(files)
     cp(i, supportdir, typesetdir)
   end
   depinstall(typesetdeps)
-  unpack()
+  unpack({".", docfiledir})
```

Comment:
The `unpack` command extracts package and related code, e.g. a `*.sty` itself, tests or documentation. The new parameter specifies directories to look into when gathering files that are to be unpacked.